playbook/app/pb_kits/playbook/tokens/width.scss

```diff
@@ -0,0 +1,20 @@
+$width_xs:      360px !default;
+$width_0:       0 !default;
+$width_none:    none !default;
+$width_sm:      540px !default;
+$width_md:      720px !default;
+$width_lg:      960px !default;
+$width_xl:      1140px !default;
+$width_xxl:     1320px !default;
+$width_100:     100% !default;
+$width_values:(
+  xs:     $width_xs,
+  0:      $width_0,
+  none:   $width_none,
+  sm:     $width_sm,
+  md:     $width_md,
+  lg:     $width_lg,
+  xl:     $width_xl,
+  xxl:    $width_xxl,
+  100:    $width_100
+);
```

Comment:
@jasperfurniss Can we discuss this? Max width is a different concept that width. These values were decided based on maximum container sizes. I feel I would need to take a look at tailwinds here to define the proper tokens.